easybuild/framework/easyconfig/templates.py

```diff
@@ -242,6 +243,10 @@ def template_constant_dict(config, ignore=None, skip_lower=None, toolchain=None)
             else:
                 deps += config.get('builddependencies', [])
 
+            # Include all toolchain deps (e.g. CUDAcore template in fosscuda)
+            if config.toolchain.tcdeps is not None:
+                deps += config.toolchain.tcdeps
```

Comment:
Using `.extend` here (and above) would actually make more sense:

```python
deps.extend(config.toolchain.tcdeps)

Reply:
Well, OK, I could do that change if you want, not that I think it makes any difference (in my own mental model += already means extend)

Reply:
With `+=` you're created a new list instance, with `.extend` you're updating the existing one.

In this context, it doesn't matter much, but it helps to have a consistent style w.r.t. updating lists.

Thanks for changing it!

Reply:
that's not correct actually. += modifies in place, and is identical to expand for lists as far as I know:
```
x = [1,2,3]
y = x
x += [4,5]
print(y)
```
specifically
```
a += b  # expands to:
a = a.__iadd__(b)
```

Reply:
Hmm, ok, then it's mostly a matter of style...